specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2363,6 +2370,52 @@
         }
       }
     },
+    "AwsCloudTrailDataConnector": {
+      "x-ms-discriminator-value": "AmazonWebServicesCloudTrail",
+      "type": "object",
+      "description": "Represents Amazon Web Services CloudTrail data connector.",
+      "allOf": [
+          {
+              "$ref": "#/definitions/DataConnector"
+          }
+      ],
+      "properties": {
+          "properties": {
+              "x-ms-client-flatten": true,
```

Comment:
@mahmoudraedhasanmsft please verify if flattening produces the model that you expect.